### PR TITLE
Updated intro lecture and added to website

### PR DESCRIPTION
**Pull request by trgardos**
Updated formatting. Added people pictures. Added GAIA policy.

I also added the intro lecture to the website so students can follow along.

### CODE REVIEW

**Comment by sladenheim** [comment]
@trgardos Can you commit the figs/people/gardos.jpg and figs/people/ladenheim.jpg?

File: ds701_book/00-Class-Introduction.qmd
Comment (sladenheim) [inline ds701_book/00-Class-Introduction.qmd:1]: Do we still want the link [https://piazza.com/class/lzrr7w98lca2vt](https://piazza.com/class/lzrr7w98lca2vt) in the Piazza slide?

**Review by sladenheim — APPROVED** [review]
I think this looks good. If you can just put everyones figure file in the repo then I think this is fine.

File: ds701_book/00-Class-Introduction.qmd
Comment (trgardos) [reply ds701_book/00-Class-Introduction.qmd:1]: Yes, we can leave it there. Just won't include the access code.

**Comment by trgardos** [comment]
> I think this looks good. If you can just put everyones figure file in the repo then I think this is fine.

Oops! good catch. added them.

**Review by sladenheim — APPROVED** [review]

**Pull request merged**